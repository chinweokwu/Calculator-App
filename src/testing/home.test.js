import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/home';

describe('Snapshot test', () => {
  test('Snapshot for Home component', () => {
    const renderedComponent = renderer.create(<Home />).toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });
});
