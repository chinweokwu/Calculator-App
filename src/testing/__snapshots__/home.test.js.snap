// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test Snapshot for Home component 1`] = `
<div
  className="home"
>
  <h1>
     Home Page
  </h1>
  <p>
     Welcome to Math Magicians Built with React
  </p>
</div>
`;

exports[`Snapshot test Snapshot for Homecomponent 1`] = `
<div
  className="quote"
>
  <div
    className="login-box"
  >
    <h3>
      
    </h3>
    <div
      className="author"
    >
      
    </div>
    <button
      onClick={[Function]}
      type="button"
    >
      <span />
      <span />
      <span />
      <span />
      Change Quotes
    </button>
  </div>
</div>
`;

exports[`Snapshot test Snapshot for Quotes component 1`] = `
<div
  className="quote"
>
  <div
    className="login-box"
  >
    <h3>
      
    </h3>
    <div
      className="author"
    >
      
    </div>
    <button
      onClick={[Function]}
      type="button"
    >
      <span />
      <span />
      <span />
      <span />
      Change Quotes
    </button>
  </div>
</div>
`;

exports[`Snapshot test Snapshot for component 1`] = `
<div
  className="quote"
>
  <div
    className="login-box"
  >
    <h3>
      
    </h3>
    <div
      className="author"
    >
      
    </div>
    <button
      onClick={[Function]}
      type="button"
    >
      <span />
      <span />
      <span />
      <span />
      Change Quotes
    </button>
  </div>
</div>
`;
